stubs/tensorflow/tensorflow/config/__init__.pyi

```diff
@@ -0,0 +1,11 @@
+from typing import NamedTuple
+
+from tensorflow.config import experimental as experimental
+
+class PhysicalDevice(NamedTuple):
+    name: str
+    device_type: str
+
+def list_physical_devices(device_type: None | str = None) -> list[PhysicalDevice]: ...
+def get_visible_devices(device_type: None | str = None) -> list[PhysicalDevice]: ...
+def set_visible_devices(devices: list[PhysicalDevice], device_type: None | str = None) -> None: ...
```

Comment:
```suggestion
def set_visible_devices(devices: list[PhysicalDevice] | PhysicalDevice, device_type: None | str = None) -> None: ...
```

Reviewing implementation it looks like [here](https://github.com/tensorflow/tensorflow/blob/0ad2ed4a1fb54e9f3a53937b5c7abbfb16fc4703/tensorflow/python/eager/context.py#L1730) the code also supports single PhysicalDevice. Mostly checked as list as input type hint is usually yellow flag (variance), but there's isinstance list in that code so tuple/other Sequence would misbehave.

Reply:
Yes, I did not try using a single PhysicalDevice, but I tried using a tuple and it does not work (it raises `ValueError: Unrecognized device`).

Reply:
Fixed in 927b79f6c